test/cpp/qps/client.h

```diff
@@ -106,9 +112,20 @@ class Client {
 
   class ClientChannelInfo {
    public:
-    ClientChannelInfo(const grpc::string& target, const ClientConfig& config)
-        : channel_(CreateTestChannel(target, config.enable_ssl())),
-          stub_(TestService::NewStub(channel_)) {}
+    ClientChannelInfo() {}
+    ClientChannelInfo(const ClientChannelInfo& i) : channel_(), stub_() {
```

Comment:
nit: you don't need to explicitly call the default constructor, even though maybe the explicitness is a good thing.